example/README.md

```diff
@@ -0,0 +1,42 @@
+# Usage Examples
+
+This document contains examples on how to launch modules with various configurations (including using optional arguments).
```

Comment:
If this example is to encompass how to run this entire OCI module, shouldn't the example show calling this entire module, rather than its submodules?

Its ok to have both, but it should clearly be separated; we support 2 ways to launch:
- the entire OCI controller module (build + init)
- OCI build / init modules separately (as in your current example)